core/domain/suggestion_services.py

```diff
@@ -902,6 +902,29 @@ def auto_reject_translation_suggestions_for_exp_ids(exp_ids: List[str]) -> None:
         suggestion_models.INVALID_STORY_REJECT_TRANSLATION_SUGGESTIONS_MSG)
 
 
+def auto_reject_translation_suggestions_for_content_ids(
```

Comment:
Can we rather name this 
```suggestion
def reject_translation_suggestions_for_content_ids(
```

I don't see why we need the `auto` we still have to run this manually. This might be also updated for the other functions like this.